OSM data program V2.ipynb

```diff
@@ -0,0 +1,1086 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9425e625",
+   "metadata": {},
+   "source": [
+    "## Importing Packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "27a9178e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyrosm import get_data, OSM\n",
```

Comment:
In python the order of import is important and could introduce some bug hard to find later due to name contamination.

General rule is:

1) stdlib comes first
2) 3rd party 
3) local packages